runtime/lua/nvim/location_list.lua

```diff
@@ -0,0 +1,22 @@
+local FixList = require('nvim._fix_list')
+
+local LocList = setmetatable({}, { __index = FixList })
+
+LocList.new = function(self, title)
+  return FixList.new(self, 'setloclist', 'getloclist', 'lopen', 'lclose', title)
```

Comment:
Why do we have a separate Lua module for this? This is an entirely new API that just adds more things for people to learn, instead of using the normal VimL functions.

Reply:
I deleted it because it was in the original PR and probably not used now.